rfcs/038-sierra-series/README.md

```diff
@@ -0,0 +1,292 @@
+entry and Constituent Unit Entry. Host Item entry links the Work to something that the Work is a part of, which could be for 
+example another Work or a serie. Constituent Unit Entry links a Work to something that the Work is composed of, 
+for example another Work or a page or chapter of the Work.
+
+There are other ways in sierra to represent Work relationships but this RFC is focusing on these 3 with the aim of defining 
+a model for representing these relationships that is flexible enough to be re-used for others.
+
+## MARC representation
+
+### Sierra series
+Sierra series are in Marc tag 490 (plus sometimes 830) of the bib record. 
+Legacy series are in tag 440 and will me migrated to 490 by collections team.
+series can have a title, a volume statement and a ISSN identifier.
+
+Based on investigation of the Sierra data done at the end of March 2021, there are 131883 record with a Sierra serie statement.
+The biggest serie is "Early European Books : Printed sources to 1700 ;" with 31956 bibs, followed by
+"ACLS Humanities E-Book." with 5118.
+[write properly]
+
+### Host Item Entry
+Host Item Entry is in 773 MARC tag. Some of the information in 773 seems to overlap with series: 
+[numbers + overlaps with series]
+
+### Constituent Unit Entry
+Constituent Unit Entry is in 774 MARC tag
+[some numbers and potential for linking with works in the library]
+
+## Proposed modeling
+The proposal is to use properties `parts` and `partOf` of the Work model that are already used to represent
+archive collections.[example of archive json] For archive collections, the website renders these as a collapsible 
+hierarchy [figure of how that json is rendered].
+Given the size of many series and Host Item Entry that would not be practical in this case, so the proposal 
+is for the frontend to choose whether to render the hierarchy base on the Work `WorkType`, which is `Collection` 
+in the case of archive Works.
+
+As series and Host Item Entries can have volumes, the proposal is to represent those as Items.
+Additionally, if a serie or Host Item Entry has an identifier (such as ISSN os ISBn or other) the proposal is 
+to mint a canonical identifier in the pipeline and to expose that identifier in the API. 
+Finally, the proposal is to model a serie statement or a Host Item Entry with a type `Work`.
+
+### Examples
+ 
+- [1074785](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1074785?lang=eng&suite=cobalt&marcData=Y) 
+This is an example of a serie statement in 490 + 830. In this example the information contained in 830 and 490 is 
+  exactly the same, so the result is just one statement. 
+```
+"partOf": [
+    {
+        "title": "Morphogenesis of the vertebrate brain",
+        "items": [
+            {
+                "title": "2",
+                "type": "Item",
+                "locations": []
+            }
+        ],
+        "type": "Work"
+    }
+]
+```
```

Comment:
Comments from @jtweed:
> - We should make it clear we dedupe if 490 + 430 are the same
> - If the partOf is from 490/830 the type should probably be Series
> - How do we support the search on the series like Encore does? ?partOf.title= ?
> - If we make the volume part of the title, how do we square that with a title keyword filter?
> - Maybe that's fine and we should be considering the title + volume combination as the thing this is partOf?

Reply:
> We should make it clear we dedupe if 490 + 430 are the same

I'm trying to find examples where they are not the same, but while I can definitely find examples where 830 doesn't have a 490, so far, if it has both they either have the same info or overlapping info so it seems to me they can be collapsed in most examples?

Reply:
> Maybe that's fine and we should be considering the title + volume combination as the thing this is partOf?

I think technically that's true, although if we collapse volume and title in one field it might be more difficult to search by title only?

Reply:
I've changed the examples to do ☝️  instead of using Items, i think it's more clear what it means

Reply:
> > We should make it clear we dedupe if 490 + 430 are the same
> 
> I'm trying to find examples where they are not the same, but while I can definitely find examples where 830 doesn't have a 490, so far, if it has both they either have the same info or overlapping info so it seems to me they can be collapsed in most examples?

Quite possibly, that would be one for Collections Info to confirm.

Reply:
> I think technically that's true, although if we collapse volume and title in one field it might be more difficult to search by title only?

This is definitely the primary consideration here. If we go for a `partOf.label` search that works like the `subjects.concepts.label` etc then the value of the label will be treated as a single token and so only works in the same volume will be returned. That's arguably correct, but it doesn't give a way to navigate up a level.

> It is true that this model makes it not clear that the `Item` is the thing that the work is part of. Ie `Morphogenesis of the vertebrate brain` could and probably does have many more volumes but they don't appear as more items because they are not what this work is part of, which is a bit confusing.
> What if we in the case of things with volumes we had two levels of `partOf`. Ie in this case:
> 
> ```
>  "partOf": [
>   {
>     "title": "Morphogenesis of the vertebrate brain; 2",
>    "partOf": [
>    {
>         "title": "Morphogenesis of the vertebrate brain",
>         "type": "Work"
>     }],
>     "type": "Work"
>   }
>  ]
> ```

I definitely wouldn't model this as items and I think if we are going to split it, something like the above would be the way to do it. However, in that example there is a duplication of the main part of the title that is problematic.

I'd be tempted to not overthink the first iteration of this and make the example above partOf a single work, that includes the volume number.

Reply:
AS I said below, I think it would make sense to collapse. The only case where I don't agree with this is when we have identifiers to present. I think in that case we would present something that is potentially misleading. As I said below, I think maybe we could include the nested `partOf` only if we have identifiers?

> However, in that example there is a duplication of the main part of the title that is problematic.
yes, it doesn't look very good, but it is technically correct I think? I couldn't think of a better way to present this

Reply:
I think let's collapse these then down into a single partOf. We do have other examples of the same identifier on multiple works. If we do split, then, yes this is the best way to treat the titles, not trying to be too clever about it.